src/pytest_plugins/consume/consume.py

```diff
@@ -133,20 +119,17 @@ def pytest_configure(config):  # noqa: D103
     it uses the modified `htmlpath` option.
     """
     input_flag = any(arg.startswith("--input") for arg in config.invocation_params.args)
-    latest_flag = config.getoption("latest_source")
-
-    if input_flag and latest_flag:
-        pytest.exit("Cannot use both `--input` and `--latest`, please select one input flag.")
-
     input_source = config.getoption("fixture_source")
 
     if input_flag and input_source == "stdin":
         config.test_cases = TestCases.from_stream(sys.stdin)
         return
 
-    if latest_flag:
-        release_base_url = "https://github.com/ethereum/execution-spec-tests/releases"
-        input_source = f"{release_base_url}/latest/download/fixtures_develop.tar.gz"
+    latest_base_url = "https://github.com/ethereum/execution-spec-tests/releases/latest/download"
+    if input_source == "latest-stable-release":
+        input_source = f"{latest_base_url}/fixtures_stable.tar.gz"
+    if input_source == "latest-develop-release":
+        input_source = f"{latest_base_url}/fixtures_develop.tar.gz"
```

Comment:
I think we should use the Github API to determine which release is the latest and then define `input_source` to the the URL that points to a specific release (i.e., a URL containing `v[0-9]+\.[0-9]+\.[0-9]+`). This has a few benefits:

1. The pytest header will display the correct release version (this is a must imho, so that you can quickly see which release was actually being executed).
2. We can display the exact release in the consume commands displayed in hiveview to help reproduce tests.
3. The extracted fixtures will be extracted to the correct cache directory.

Another approach would be to grab the version from the `.meta/fixtures.ini` file, but I thinkI prefer resolving `latest`. If we're being very careful, we could check that the resolved version matches that of `.meta/fixtures.ini`, but it's a bit paranoid.

Reply:
Btw, there's an implementation of resolving `latest` on the command-line in [our docs](https://ethereum.github.io/execution-spec-tests/v3.0.0/consuming_tests/#obtaining-the-most-recent-release-artifacts) - but we should add a pure python implementation here instead of relying on `jq` (and potentially update the docs).